### PR TITLE
KNOX-2813 Fix a url proxy of Hiveserver2 UI

### PR DESCRIPTION
**Pull request by Flamingo93**
## What changes were proposed in this pull request?

Fix a url proxy of Hiveserver2 UI

## How was this patch tested?
UI test
Before fixed
image

After fixed
image

### CODE REVIEW

**Review by smolnar82 — APPROVED** [review]
LGTM
Let's wait until Travis finishes an we can merge.
@moresandeep - Could you please take a look? Thanks!

**Comment by Flamingo93** [comment]
Hi @moresandeep Could you please take a look? :）

**Pull request merged**
